superset-frontend/src/dashboard/components/SliceHeaderControls/index.tsx

```diff
@@ -261,10 +340,23 @@ const ViewResultsModalTrigger = ({
 };
 
 const SliceHeaderControls = (props: SliceHeaderControlsPropsWithRouter) => {
+  const [dropdownIsOpen, setDropdownIsOpen] = useState(false);
+  const [tableModalIsOpen, setTableModalIsOpen] = useState(false);
+  const [drillModalIsOpen, setDrillModalIsOpen] = useState(false);
+  const [selectedKeys, setSelectedKeys] = useState<string[]>([]);
   const [openScopingModal, scopingModal] = useCrossFiltersScopingModal(
     props.slice.slice_id,
   );
 
+  const queryMenuRef: RefObject<any> = useRef(null);
+  const menuRef: RefObject<any> = useRef(null);
+
+  const toggleDropdown = ({ close }: { close?: boolean } = {}) => {
```

Comment:
Why an object in args? Won't `(close: boolean) => {...}` work the same?

Reply:
Yes, you're right, the functionality is the same. It's so that when you call `toggleDropdown({ close: true });` it's more explicit than `toggleDropdown(true);`